compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -74,6 +80,67 @@ int entry(int argc, char **argv)
     setenv("LUCI_LOG", "100", 0);
   }
 
-  // TODO Run Bisect method
+  auto data_path = arser.get<std::string>("--data");
+  auto input_model_path = arser.get<std::string>("--input_model");
+  auto output_model_path = arser.get<std::string>("--output_model");
+
+  float qerror_ratio = arser.get<float>("--qerror_ratio");
+  if (qerror_ratio < 0.0 || qerror_ratio > 1.f)
+  {
+    std::cerr << "ERROR: quantization ratio should be above 0 and below 1" << std::endl;
+    return EXIT_FAILURE;
+  }
+  auto start = std::chrono::high_resolution_clock::now();
+
+  using namespace mpqsolver::bisection;
+
+  // optimize
+  mpqsolver::bisection::BisectionSolver solver(data_path, qerror_ratio);
+  {
+    auto value = arser.get<std::string>(bisection_str);
+    if (value == "auto")
+    {
+      // TODO
+      std::cerr << "Not implemented yet " << input_model_path << std::endl;
+      return EXIT_FAILURE;
```

Comment:
Why `auto` is not implemented?

Reply:
To reduce commit size it will be introduced in the next PR.